tsdate/core.py

```diff
@@ -523,6 +524,78 @@ def scale_geometric(self, fraction, value):
         return fraction * value
 
 
+class VariationalLikelihoods:
+    """
+    A class to store and process likelihoods for use in variational inference.
```

Comment:
I think this stores the gamma in Laplace space, right? So that we can do the convolutions by addition. Should we say that up top?

Reply:
Not in Laplace space-- "exponential family parameter" space (e.g. if you multiply / divide two gamma pdfs, that's the same as adding/subtracting their natural parameters). I can add some description

Reply:
That would be great, thanks Nate.